bash/atomic

```diff
@@ -311,9 +311,19 @@ _atomic_pull() {
 }
 
 _atomic_ps() {
+	local all_options="
+		--all -a
+		--filter -f
+		--help
```

Comment:
Not sure how easy this would be to do, but have command completion on filterables.
+        filterables = ["container", "image", "command", "created", "status", "runtime"]